src/components/Debugger/Timeline.tsx

```diff
@@ -264,6 +264,18 @@ const getPageUrl = (entries: Entry[]) => {
   return page ? new URL(page.value) : null;
 };
 
+const extractNetworkUserId = (cookies: Cookie[]): Cookie | undefined => {
+  // consider only cookies with the UUID format
+  const uuidRegexp =
+    /^[0-9a-fA-F]{8}\b-[0-9a-fA-F]{4}\b-[0-9a-fA-F]{4}\b-[0-9a-fA-F]{4}\b-[0-9a-fA-F]{12}$/gi;
```

Comment:
This seems a bit overkill; don't think we need the `\b` matches at all, nor global flag, and case insensitivity is moot when we're already checking both cases explicitly.

Reply:
Good point! Have used a simpler regular expression.